docs/library/zephyr.zsensor.rst

```diff
@@ -0,0 +1,123 @@
+.. currentmodule:: zsensor
+
+:mod:`zsensor` --- Zephyr sensor bindings
+=========================================
+
+.. module:: zsensor
+    :synopsis: zephyr sensor bindings
+
+The ``zsensor`` module contains a class for using sensors with Zephyr.
+
+.. _zsensor.Sensor:
+
+class Sensor --- sensor control for the Zephyr port
+---------------------------------------------------
+
+Use this class to access data from sensors on your board.
+See Zephyr documentation for sensor usage here: `Sensors
+<https://docs.zephyrproject.org/latest/reference/peripherals/sensor.html?highlight=sensor#sensors>`_.
+
+Sensors are defined in the Zephyr devicetree for each board. The quantities that a given sensor can
+measure are called a sensor channels. Sensors can have multiple channels to represent different axes
+of one property or different properties a sensor can measure. See `Channels`_ below for defined sensor
+channels.
+
+Constructor
+~~~~~~~~~~~
+
+.. class:: Sensor(device_name)
+
+    Device names are defined in the devicetree for your board.
+    For example, the device name for the accelerometer in the FRDM-k64f board is "FXOS8700".
+
+Methods
+~~~~~~~
+
+.. method:: Sensor.measure()
+
+    Obtains a measurement sample from the sensor device using Zephyr sensor_sample_fetch and
+    stores it in an internal driver buffer as a useful value, a tuple of {integer part of value,
+    fractional part of value in 1-millionths}.
```

Comment:
It might be confusing using braces `{..}` for a "tuple" in this context (because in Python `{}` is used for dict/set, and tuple means `(a, b, ...)`).  Maybe call it "pair" here and use normal parenthesis `()`?